cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/BasicPushStrategy.java

```diff
@@ -74,25 +75,22 @@
 
     private static JSONHelper sJsonHelper = new JSONHelper();
 
-    public BasicPushStrategy(PushReplication pushReplication) {
-        this(pushReplication, null);
-    }
+    public int changeLimitPerBatch = 500;
```

Comment:
Is there a reason for this `changeLimitPerBatch` to be different from the one in [BasicPullStragtegy](https://github.com/cloudant/sync-android/pull/222/files#diff-75ce3c6e59410e385e6b519bfe3e9824R81)?